sdk/spring/azure-spring-data-cosmos/src/test/java/com/azure/spring/data/cosmos/domain/BasicItem.java

```diff
@@ -0,0 +1,51 @@
+package com.azure.spring.data.cosmos.domain;
```

Comment:
Add the following at the top of all new files, which can be found in any file to copy/paste:

// Copyright (c) Microsoft Corporation. All rights reserved.
// Licensed under the MIT License.

Reply:
Ah, is that why builds are failing on this file? Ok, will do, thanks!